curvefs/src/mds/fs_manager.cpp

```diff
@@ -1179,5 +1180,29 @@ bool FsManager::FillVolumeInfo(common::Volume* volume) {
     return true;
 }
 
+void FsManager::UpdateFsUsage(const FsId2FsUsage& infos) {
+    for (auto const& fsUsage : infos) {
+        FsInfoWrapper wrapper;
```

Comment:
seems no need to persist usage in fsinfo

Reply:
> seems no need to persist usage in fsinfo

fix